pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2552,6 +2598,14 @@ func (s *AWSCloud) EnsureLoadBalancerDeleted(service *api.Service) error {
 		}
 	}
 
+	{
+		// Remove the DNS cname if defined
+		err := s.removeLoadBalancerCnameEntry(service, lb.DNSName)
+		if err != nil {
+			glog.Warningf("Error removing cname entry for %v: %v", service.Name, err)
```

Comment:
K8s already spams the logs with a lot of errors that don't have any user impact. Where as this would leave a CName in R53 it doesn't stop any application working hence we went with a warning.